src/button/css/objects/_honeycomb.button.objects.buttons.scss

```diff
@@ -28,10 +28,10 @@
 
     &:hover {
       background-color: $button-hover-background !important;
-      color: $button-hover-color !important;
+      color: $button-hover-colour !important;
```

Comment:
This is a really minor thing, but we should probably be consistent and just adopt the US spelling everywhere, otherwise we end up with a mis-match.